src/aktualizr.h

```diff
@@ -5,12 +5,12 @@
 
 class Aktualizr : public boost::noncopyable {
  public:
-  Aktualizr(const Config& config);
+  Aktualizr(Config& config);
```

Comment:
Why You removed const?

Reply:
I think we should eventually move it from Config, it doesn't seem to belong there especially as we'll have even move discovery mechanisms in aktualizr